com.ibm.wala.core/src/com/ibm/wala/analysis/typeInference/TypeInference.java

```diff
@@ -727,8 +727,8 @@ public void visitStoreIndirect(SSAStoreIndirectInstruction instruction) {
 
     @Override
     public IVariable makeVariable(int valueNumber) {
-      SymbolTable st = ir.getSymbolTable();
       if (doPrimitives) {
+        SymbolTable st = ir.getSymbolTable();
```

Comment:
@msridhar Done with dccae97de3079f35cfb9f6591f9d24f3931a7a9c and cd013804fc90c9dbb0832684d850d37656db9d26.